include/cetl/any.hpp

```diff
@@ -105,46 +262,35 @@ template <std::size_t Footprint,
           bool        Copyable  = true,
           bool        Movable   = Copyable,
           std::size_t Alignment = alignof(std::max_align_t)>
-class any : private detail::base_move<Footprint, Copyable, Movable, Alignment>
+class any : detail::base_move<Footprint, Copyable, Movable, Alignment>
 {
 public:
-    constexpr any() noexcept = default;
-
-    any(const any& other)
-    {
-        if (other.has_value())
-        {
-            other.handle(detail::action::Copy, this);
-        }
-    }
-
-    any(any&& other) noexcept
-    {
-        if (other.has_value())
-        {
-            other.handle(detail::action::Move, this);
-        }
-    }
+    constexpr any()           = default;
+    any(const any& other)     = default;
+    any(any&& other) noexcept = default;
 
     template <
         typename ValueType,
         typename Tp = std::decay_t<ValueType>,
         typename = std::enable_if_t<!std::is_same<Tp, any>::value && !pf17::detail::is_in_place_type<ValueType>::value>>
     any(ValueType&& value)
     {
-        soo_handler<Tp>::create(*this, std::forward<ValueType>(value));
+        base::template make_handlers<Tp>();
+        new (base::get_raw_storage()) Tp(std::forward<ValueType>(value));
```

Comment:
The pattern with `make_handlers` and then placement `new` repeats a lot here; should we extract it into a method?